src/sentry/models/projectownership.py

```diff
@@ -118,6 +120,10 @@ def get_owners(
         rules = cls._matching_ownership_rules(ownership, project_id, data)
 
         if not rules:
+            project = Project.objects.get(id=project_id)
+            if features.has("organizations:issue-alert-fallback-targeting", project.organization):
+                return [], None
```

Comment:
I don't think we need this if we change determine_eligible_recipients to stop calling this function